_docs/durability.md

```diff
@@ -4,4 +4,138 @@ title: Durability
 permalink: /docs/durability/
 ---
 
-**Note:** The original durability implementation had a variety of shortcomings and has been replaced by a more robust and performant implementation as discussed at [issue 198](https://github.com/cerner/clara-rules/issues/198).  Documentation on this new durability API is pending.
+### Overview ###
+
+The [clara.rules.durability]({{site.clojuredoc}}clara.rules.durability.html) namespace offers functions serialize and deserialize Clara sessions.  The motivation and goals of this durability layer is described in [issue 198](https://github.com/cerner/clara-rules/issues/198).
```

Comment:
grammar: "offers functions serialize and deserialize" should be "offers functions that" or have some other connective word/phrase.